tt_eager/tt_dnn/op_library/eltwise_binary/eltwise_binary_op.hpp

```diff
@@ -136,7 +136,8 @@ struct make_eltwise_binary {
                     (in_a.get_legacy_shape() == in_b.get_legacy_shape()) or
                     (in_a.get_legacy_shape().without_padding() == in_b.get_legacy_shape().without_padding()),
                     "Input shapes must be the same!");
-                return operation::run_with_autoformat(
+                return operation::run_without_autoformat(
```

Comment:
Can you use `operation::run`? 


`operation::run_without_autoformat` still tries to move the tensor to the device which is an auto format

Reply:
I did try `operation::run` initially , but it causes  **Operands to eltwise binary need to be on device!** error for various ops. 

_E       RuntimeError: TT_FATAL @ tt_eager/tt_dnn/op_library/eltwise_binary/eltwise_binary_op.cpp:124: input_tensor_a.storage_type() == StorageType::DEVICE and input_tensor_b.storage_type() == StorageType::DEVICE
E       info:
E       Operands to eltwise binary need to be on device!_

Reply:
I was able to replace `autoformat` operations and work with `operation::launch_op`, `operation::run` instead. Except for `prod_bw` op and two cpp tests - test_raw_host_memory_pointer, tensors/test_async_tensor_apis the CI passes for all other ops and tests with fixes in this PR. 
CI : https://github.com/tenstorrent/tt-metal/actions/runs/9086023826